src/js/components/Avatar/doc.js

```diff
@@ -68,4 +68,34 @@ export const themeDoc = {
     type: 'number',
     defaultValue: undefined,
   },
+  'avatar.text.size.xsmall': {
+    description:
+      'The xsmall size of the text in Avatar (Mapped according Avatar size).',
```

Comment:
```suggestion
      "The size of the text that is mapped according to 'avatar.size.xsmall'". ,
```
What do you think of the proposed structure sentence? Should we add to it anything so it will be more clear?

Reply:
At first I was a little bit worried about people getting confused about the relation being medium to large, large to xlarge and so on. But considering the docs show the default value it's all good. And I like the suggested change. Will apply it in the next commit! Thanks for that.